android/src/main/java/com/rnmapbox/rnmbx/components/styles/sources/RNMBXRasterDemSourceManager.kt

```diff
@@ -27,4 +30,14 @@ class RNMBXRasterDemSourceManager(private val mContext: ReactApplicationContext)
         const val LOG_TAG = "RNMBXRasterDemSourceManager"
         const val REACT_CLASS = "RNMBXRasterDemSource"
     }
+
+    @ReactProp(name = "existing")
+    override fun setExisting(view: RNMBXRasterDemSource, value: Dynamic) {
+        view.mExisting = value.asBoolean()
+    }
+
+    @ReactProp(name = "tileSize")
+    override fun setTileSize(view: RNMBXRasterDemSource, value: Dynamic) {
+        // TODO: it does not exist on this class
```

Comment:
we can log an error, that it's not implemented.
```suggestion
       Logger.e("RNMBXRasterDemSourceManager", "tileSize not implemented")
```